solutions/1/style.css

```
@@ -0,0 +1,57 @@
+body{
+	height: 100vh;
+	background-color: #f1f1f1;
+	font-size: 75px;
+	overflow: hidden;
+}
+
+body, p{
+	margin: 0 !important;
+	padding: 0 !important;
+}
+
+.container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: inherit;
+}
+
+.circle{
+	z-index: 999;
```

Не обязательно такое большое значение z-index.
```suggestion
	z-index: 1;
```

---

А ещё это свойство в принципе не работает без указания position со значением, отличным от дефолтного `static`.